main-packages-list.txt

```diff
@@ -22,4 +22,3 @@ python3-scciclient
 python3-sushy
 python3-sushy-oem-idrac
 qemu-img
-syslinux-nonlinux
```

Comment:
I wonder if we can either filter this in the `cat /tmp/main-packages-list.txt` in the dockerfile, or add the `-x` option to exclude the package only when  `uname -m` tells us it's ppc64?